Sources/WebSocketKit/WebSocketHandler.swift

```diff
@@ -90,7 +108,17 @@ extension WebSocket {
         with config: Configuration,
         onUpgrade: @Sendable @escaping (WebSocket) -> ()
     ) -> EventLoopFuture<Void> {
-        let webSocket = WebSocket(channel: channel, type: type)
+        
+        let webSocket =  config.deflateConfig != nil ?
+        WebSocket(channel: channel,
+                  type: type,
+                  pmce: PMCE(config:config.deflateConfig!,
+                             channel: channel,
+                             socketType: type)) :
+        WebSocket(channel: channel,
+                  type: type)
+        
+        print("websocket-kit: WebSocket.configure(on:...) created websocket \(webSocket) \nof \(type) \nwith \(config)")
```

Comment:
I think logging whether deflate was enabled _could_ be useful in an info message. But not a print

Reply:
Also consider using `#function` to get the current function this log was produced in

Reply:
these are great suggestions, I had suffered a lack of the logger for a while.